### PR TITLE
Fixes #6112 - Updates SSH-Remoting-in-PowerShell-Core

### LINKED ISSUE #6112: Update SSH article to parallel WinRM article
The WinRM Security article includes information about the security provided by WinRM. We need to update this article with similar information.

Also worth pointing out that we don't support remoting endpoints with SSH. So you can't do things like JEA.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 85e79ea6-1d79-6147-aa3e-362b04bc7528
* Version Independent ID: 8877b05f-1f52-8c75-2d7d-5100cc52c4cb
* Content: [PowerShell Remoting Over SSH - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core?view=powershell-7)
* Content Source: [reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary

Updates SSH Remoting in PowerShell Core for multiple information items

## PR Context

Fixes #6112 
Fixes [AB#1733550](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1733550)

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [x] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
    Optionally, enable key authentication:
 
-   ```
+   ```Output
```
Comment (sdwheeler) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:108]: This is not output. This is another setting within the configuration file.

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 2]
```diff
@@ -71,14 +71,14 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
...
    PasswordAuthentication yes
    ```
 
    Create the SSH subsystem that hosts a PowerShell process on the remote computer:
 
-   ```
-   Subsystem powershell c:/progra~1/powershell/7/pwsh.exe -sshs -NoLogo -NoProfile
+   ```powershell
```
Comment (sdwheeler) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:80]: This is a configuration setting in a file, not PowerShell.

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 3]
```diff
@@ -22,13 +22,13 @@ JEA.
 The `New-PSSession`, `Enter-PSSession`, and `Invoke-Command` cmdlets now have a new parameter set to
 support this new remoting connection.
 
-```
+```powershell
```
Comment (sdwheeler) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:25]: This is syntax not PowerShell.

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md [hunk 4]
```diff
@@ -71,14 +71,14 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
    Make sure password authentication is enabled:
 
-   ```
+   ```Output
```
Comment (sdwheeler) [inline reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:74]: This is a configuration setting in a file

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md (hunk 3 above)
Comment (chasewilson) [reply reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:25]: fixed

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md (hunk 4 above)
Comment (chasewilson) [reply reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:74]: dixed

File: reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md (hunk 1 above)
Comment (chasewilson) [reply reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md:108]: Fixed

**Pull request merged**